joomla/joomla-cms
# [4.1] Images in the Smart Search results

### Summary of Changes
The PR allows the use of images in the finder's results.

A new setting was added named "Result Image" both in the com_finder's (aka Smart Search) configuration and it's respective "search" menu item.


### Testing Instructions
1. Go to the "Smart Search" component, press the "Clear Index" button and then the "Index".
2. Press the "Options" button to load the configuration and enable the "Result Image" setting.
3. Use the smart search in the site's front-end and check the returned articles that have "Intro Image" set.


### Actual result BEFORE applying this Pull Request

![Smart_Search_1](https://user-images.githubusercontent.com/1838041/108255744-54023b00-7165-11eb-8fba-c727b7272119.png)


### Expected result AFTER applying this Pull Request
![Smart_Search2](https://user-images.githubusercontent.com/1838041/108255768-582e5880-7165-11eb-97f6-ccb7286ddd49.png)
### Sidenotes

Atm only the _com_content_ finder plugin uses the images API function. Support for the rest plugins will be added in a next PR.
No styling is added, though a new setting (Image class) is introduced, according to the setting of com_content's config.
If anyone thinks that additional styling should be applied, plz commit.

### Documentation Changes Required
DKN
